source/release-notes.txt

```diff
@@ -10,6 +10,29 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.45.4
```

Comment:
We should also update the snooty.toml release version as part of these changes.

Reply:
Fixed